<launch>
    <arg name="pkg_name" value="jetbot_ros"/>
    <arg name="node_name" default="wheels_driver_node"/>
    <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" clear_params="true" required="true"/>
</launch>
